pianoPlayer/main.js

```diff
@@ -0,0 +1,108 @@
+/* ISSUES & TODOs
+- add music!!
+- stop propagation / capture=true for when the keys are pressed (excluding the letters!)
+- mouseup event not caught when mouseout before
+- prevent key from overlapping the dark keys when it's being pressed
+- fix skeuomorphic design for upper keys
+*/
+
+// The keys and notes variables store the piano keys
+// plucks out the corresponding DOM elements and stores them as such
+const keys = ['c-key', 'd-key', 'e-key', 'f-key', 'g-key', 'a-key', 'b-key', 'high-c-key', 'c-sharp-key', 'd-sharp-key', 'f-sharp-key', 'g-sharp-key', 'a-sharp-key'];
+const notes = [];
+keys.forEach(function (key) {
+    notes.push(document.getElementById(key));
+})
+
+// simple skeuomorphic design for when a key is played
+function keyPlay(event) {
+    event.target.style.backgroundColor = 'darkgrey';
+    event.target.style.position = 'relative';
+    event.target.style.top = '5px';
+    event.target.style.left = '5px';
+}
+function keyReturn(event) {
+    event.target.style.backgroundColor = '';
+    event.target.style.position = '';
+    event.target.style.top = '';
+    event.target.style.left = '';
+}
+function playNote(note) {
+    note.addEventListener('mousedown', keyPlay);
+    note.addEventListener('mouseup', keyReturn);
+}
+
+// loop that assigns the eventListeners to each key DOM element
+notes.forEach(playNote);
+
+// These variables store the buttons that progress the user through the lyrics
+let nextOne = document.getElementById('first-next-line');
+let nextTwo = document.getElementById('second-next-line');
+let nextThree = document.getElementById('third-next-line');
+let startOver = document.getElementById('fourth-next-line');
+
+// This variable stores the '-END' lyric element
+let lastLyric = document.getElementById('column-optional');
```

Comment:
Why are these `let`s? `let` is when you define a variable and want to be able to change it. These should be `const`, which are defined once and never change.

When I read `let`, I'm now carrying a little reminder in my head to watch out for how this will change over time. With `const` it's like "she just wants this value/reference, and I'm not on the lookout for when it will change"

Reply:
Codecademy is heavy on `let`. I need to really train myself to use more `const`..